src/index.js

```diff
@@ -91,3 +95,5 @@ app.use(function(err, req, res, next) {
 const server = app.listen(process.env.PORT || 3000, function() {
     console.log("Listening on port " + server.address().port);
 });
+
+module.exports = server;
```

Comment:
Good job @muchristian, but you don't have to create a new server here, you could remove 
`const server = ` and export app like this `module.exports = app;`